pkg/controller/storagecluster/reconcile.go

```diff
@@ -1253,13 +1203,13 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 	const purgeOSDScript = `
 set -x
 
-osd_status=$(ceph osd tree | grep "osd.${FAILED_OSD_ID} " | awk '{print $5}') 
-if [[ "$osd_status" == "up" ]]; then 
+osd_status=$(ceph osd tree | grep "osd.${FAILED_OSD_ID} " | awk '{print $5}')
+if [[ "$osd_status" == "up" ]]; then
   echo "OSD ${FAILED_OSD_ID} is up and running."
   echo "Please check if you entered correct ID of failed osd!"
-else 
+else
   echo "OSD ${FAILED_OSD_ID} is down. Proceeding to mark out and purge"
-  ceph osd out osd.${FAILED_OSD_ID} 
+  ceph osd out osd.${FAILED_OSD_ID}
```

Comment:
Even here, these changes are not required!

Reply:
This is correct IMO as the previous contributor **added extra spaces**, my vscode settings remove unnecessary extra whitespace. I didn't add/removed any spaces by myself.